rector.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ]);
+
+    $rectorConfig->sets([
+        \Rector\PHPUnit\Set\PHPUnitSetList::PHPUNIT_100,
```

Comment:
Isn't this too aggressive? We're using 9

Reply:
Actually Rector itself suggets to use it

>     /**
>     * @deprecated Use PHPUnitSetList::PHPUNIT_100 directly
>     * @var string
>     */
>    public const PHPUNIT_91 = __DIR__ . '/../../config/sets/phpunit91.php';